src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php

```diff
@@ -114,13 +117,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $builder->setTimeout(null);
         $process = $builder->getProcess();
 
-        if (OutputInterface::VERBOSITY_VERBOSE > $output->getVerbosity()) {
-            $process->disableOutput();
+        try {
+            if (OutputInterface::VERBOSITY_NORMAL > $output->getVerbosity()) {
+                $process->disableOutput();
```

Comment:
the try/cache can be inside the ``else`` (and defining ``$callback = null;`` before the ``if``)